packages/desktop-client/src/components/modals/GoalTemplateModal.tsx

```diff
@@ -0,0 +1,88 @@
+import { theme } from '../../style';
+import { Link } from '../common/Link';
+import { Modal, ModalCloseButton, ModalHeader } from '../common/Modal';
+import { Text } from '../common/Text';
+import { View } from '../common/View';
+
+export function GoalTemplateModal() {
+  return (
+    <Modal name="goal-templates">
+      {({ state: { close } }) => (
+        <>
+          <ModalHeader
+            title="Goal Templates"
+            rightContent={<ModalCloseButton onPress={close} />}
+          />
+          <View
+            style={{
+              flexDirection: 'row',
+              fontSize: 13,
+            }}
+          >
+            <Text style={{ textAlign: 'left' }}>
+              <h3>Weekly:</h3>
+              <strong>
+                #template $10 repeat every week starting 2025-01-03
+              </strong>{' '}
+              - Budget $10 a week <br />
```

Comment:
We'll want to translate all of this text using  'react-i18next';

We have the <Trans> component and the useTranslation hook for it

Reply:
not super sure how to test this but will initializing and importing useTranslation, importing i18next, and simply wrapping the string in {t('')} 
such as 
```
import { useTranslation } from 'react-i18next';
const { t } = useTranslation();
{t('#template $10 repeat every week starting 2025-01-03')}
```

Reply:
You can use ```t('text to translate')``` if you need to use it outside of the render, or you can use the Trans component ```<Trans>text to translate</Trans>``` within the render.

You can test it by running ```yarn generate:i18n``` and checking the values the values in the generated json.